github/github.go

```diff
@@ -109,9 +109,6 @@ const (
 	// https://developer.github.com/changes/2018-12-18-interactions-preview/
 	mediaTypeInteractionRestrictionsPreview = "application/vnd.github.sombra-preview+json"
 
```

Comment:
Shouldn't lines 49-51 in github.go be removed too?

Reply:
Yes, I missed this one